public/app/plugins/datasource/loki/language_provider.ts

```diff
@@ -177,6 +177,7 @@ export default class LokiLanguageProvider extends LanguageProvider {
     return queries.map(query => ({
```

Comment:
This is my fault, we shouldn't introduce Grafanaish concepts like refId and key here, right @davkal? That's why in the original code (https://github.com/grafana/grafana/blob/a21f6777c16bca7f8c15ef7568f6095ff31a45e9/public/app/features/explore/state/actions.ts#L346) used ...generateEmptyQuery(i). So refId and key should be removed and tests updated.